x-pack/plugins/security_solution/server/endpoint/routes/resolver/queries/related_events.ts

```diff
@@ -0,0 +1,89 @@
+/*
```

Comment:
This file was really a move: `event.s -> related_events.ts`

Reply:
The files moved to their `related_events.ts` counterparts are essentially `deprecated` right?

Reply:
That's correct. I wanted the `events.ts` file to have the new implementation since that name seemed most appropriate.

Reply:
Sweet, good to know. Thanks. We can probably `@deprecated` these files